### PR TITLE
New Footer

### LINKED ISSUE #4: Footer component
image

### PR DESCRIPTION
**Pull request by Rofy87**

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  FaPinterestP,
+  FaInstagram,
+  FaLinkedin,
+} from "react-icons/fa";
+import logo from "./logo.png";
+import { NavLink } from "react-router-dom";
+import dog from "./Bg.png";
+import "./Footer.css";
+
+// footer component start here 👌
```
Comment (halitbatur) [inline src/components/Footer/Footer.js]: lets delete this comment

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+footer {
```
Comment (halitbatur) [inline src/components/Footer/Footer.css:1]: this is duplicated

File: src/components/Footer/Footer.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.headersColor {
+  color: #ff7777;
+  text-align: left;
+}
+
+.ulJustify {
+  text-align: left;
+}
```
Comment (halitbatur) [inline src/components/Footer/Footer.css:18]: new line

File: src/components/Footer/Footer.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+footer {
+  background-color: #31393c;
+  color: white;
+}
+
+.dogImage {
+  width: 30;
+  height: 20;
+  border-radius: 5px;
+}
```
Comment (halitbatur) [inline src/components/Footer/Footer.css:29]: new line

**Review by halitbatur — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (Rofy87) [reply src/components/Footer/Footer.js]: Done

File: src/components/Footer/Footer.css (hunk 2 above)
Comment (Rofy87) [reply src/components/Footer/Footer.css:1]: Done

File: src/components/Footer/Footer.css (hunk 3 above)
Comment (Rofy87) [reply src/components/Footer/Footer.css:18]: Done

File: src/components/Footer/Footer.css (hunk 4 above)
Comment (Rofy87) [reply src/components/Footer/Footer.css:29]: Done

**Review by halitbatur — APPROVED** [review]
Looks good to me good work @Rofy87

**Review by mustafaguldag — APPROVED** [review]
Hi Rahaf, it looks good to me. It passes all tests and code seems good to me. Thanks for contribute!

**Pull request merged**